dspace/modules/api/src/main/java/org/dspace/JournalUtils.java

```diff
@@ -605,17 +605,19 @@ public static int compareItemAuthorsToManuscript(Item item, Manuscript manuscrip
         DCValue[] itemAuthors = item.getMetadata("dc", "contributor", "author", Item.ANY);
         result.append("item " + item.getID() + " has " + itemAuthors.length + " authors while manuscript has " + manuscript.getAuthorList().size() + " authors\n");
         for (int j = 0; j < itemAuthors.length; j++) {
-            for (Author a : manuscript.getAuthorList()) {
-                double score = JournalUtils.getHamrScore(Author.normalizeName(itemAuthors[j].value).toLowerCase(), a.getNormalizedFullName().toLowerCase());
-                result.append("author " + itemAuthors[j].value + " matched " + a.getUnicodeFullName() + " with a score of " + score + "\n");
-                if (score > 0.6) {
+            Author itemAuthor = new Author(itemAuthors[j].value);
+            for (Author msAuthor : manuscript.getAuthorList()) {
+                double score = JournalUtils.getHamrScore(Author.normalizeName(itemAuthors[j].value).toLowerCase(), msAuthor.getNormalizedFullName().toLowerCase());
```

Comment:
Why compute the score if it is not being used as part of the matching criteria?

Reply:
It's still useful for me to see what the score value is, for debugging purposes.